tools/c7n_gcp/c7n_gcp/resources/sql.py

```diff
@@ -152,6 +152,34 @@ def get_resource_params(self, model, resource):
             }
         }
 
+@SqlInstance.action_registry.register('set-high-availability')
+class SqlInstanceHighAvailability(MethodAction):
+
+    schema = type_schema(
+        'set-high-availability',
+        value={'type': 'boolean'})
+    method_spec = {'op': 'patch'}
+    path_param_re = re.compile('.*?/projects/(.*?)/instances/(.*)')
+    method_perm = 'update'
+
+    def get_resource_params(self, model, resource):
+        if str(self.data.get('value')) == 'False':
```

Comment:
str conversion to boolean comparison is quite odd. 

I would suggest making value required in the schema (`required=True`) and then `if self.data['value'] is False` for an identity comparison.

Reply:
fixed the comment